Add splitter instructions for leader flow (#111)
ObolNetwork/obol-docs: docs/int/quickstart/group/quickstart-group-leader-creator.md

      <li>Enter the addresses and split percentages for the <code>Rewards</code> to receive staking rewards, MEV and priority fees.</li></ul><br></br>

      ![Create Group](/img/Guide03-splitter.png)

      </TabItem>
    </Tabs>

    - Create Cluster Configuration

6. Review, Confirm and Sign the following with your wallet

<Tabs groupId="leader-creator">
  <TabItem value="leader" label="Leader" default>
    <ul>
    <Tabs groupId="withdrawl-method">
    <TabItem value="single" label="Single Address">
    Confirm the config and Sign
    <ul>
    <li>The config hash. This is a hashed representation of all of the details for this cluster.</li>
    <li>The <code>operator_config_hash</code></li>
    <li>Your <code>ENR</code>: this signature authorises the key represented by this ENR to act on your behalf in the cluster.</li>
    </ul>
    </TabItem>
    <TabItem value="splitter" label="Splitter Contract">
    <ul><li>Deploy the withdrawal contracts by signing the transactions in your wallet.</li></ul>
    <ul><li>Confirm the config and Sign</li></ul>
    <ul><ul>
    <li>The config hash. This is a hashed representation of all of the details for this cluster.</li>

>     <li>The <code>operator_config_hash</code></li>

Reviewer: ```suggestion
     <li>The <code>operator_config_hash</code>. This is your acceptance of the terms as a participating node operator.</li>
```

>     <li>Your <code>ENR</code>: this signature authorises the key represented by this ENR to act on your behalf in the cluster.</li>

Reviewer: ```suggestion
    <li>Your <code>ENR</code>. Signing your ENR authorises the corresponding private key to act on your behalf in the cluster.</li>
```

>     </ul></ul>

Reviewer: This probably matches the previous <ul><ul>, should probably be only one of these on each side

>     </TabItem></Tabs></ul>

Reviewer: Does this </ul> match anything? Can't figure it out on github rn.

  </TabItem>
  <TabItem value="creator" label="Creator">
    <ul>
    <Tabs groupId="withdrawl-method">
    <TabItem value="single" label="Single Address">

>     Confirm the config and Sign

Reviewer: ```suggestion
    You will be asked to confirm your configuration and to sign:
```

>     <ul><li>The config hash. This is a hashed representation of all of the details for this cluster.</li></ul>

Reviewer: ```suggestion
    <ul><li>The `config_hash`. This is a hashed representation of the details of this cluster, to ensure everyone is agreeing to an identical setup.</li></ul>
```

    </TabItem>

>     <TabItem value="splitter" label="Splitter Contract">

Reviewer: ```suggestion
    <TabItem value="splitter" label="Reward Splitter">
```

>     <ul><li>Deploy the withdrawal contracts by signing the transactions in your wallet.</li></ul>

Reviewer: ```suggestion
    <ul><li>Deploy the withdrawal manager contracts by signing the transactions in your wallet.</li></ul>
```

>     <ul><li>Confirm the config and Sign</li></ul>

Reviewer: ```suggestion
    <ul><li>You will be asked to confirm your configuration and to sign:</li></ul>
```

>     <ul><ul>

Reviewer: Same weird double `<ul><ul>` here

>     <li>The config hash. This is a hashed representation of all of the details for this cluster.</li>

Reviewer: ```suggestion
    <ul><li>The `config_hash`. This is a hashed representation of the details of this cluster, to ensure everyone is agreeing to an identical setup.</li></ul>
```

>     </ul></ul>

Reviewer: Probably shouldn't double.

>     </TabItem></Tabs></ul>

Reviewer: does this trailing `</ul>` have a match?
